E1_TFHub_Sample_Deploy/export_test.py

```diff
@@ -0,0 +1,50 @@
+import tensorflow as tf
+import numpy as np
+import tensorflow_datasets as tfds
+import tensorflow_hub as hub
+import unittest
+import argparse
+
+FLAGS = None
+
+class TFHubMNISTTest(tf.test.TestCase):
+  def setUp(self):
+    kwargs = {}
+    if FLAGS.data_dir:
+      kwargs["data_dir"] = FLAGS.data_dir
+    self.dataset = tfds.load("mnist",
+                             split="test",
+                             **kwargs).shuffle(FLAGS.buffer_size,
+                                               reshuffle_each_iteration=True).batch(FLAGS.batch_size)
```

Comment:
The formatting seems weird here. Can you run these files through the python linter?

Reply:
I'm linting with autopep8. Let me run through pylint and check